src/main/kotlin/org/kamiblue/client/module/modules/render/Xray.kt

```diff
@@ -0,0 +1,33 @@
+package org.kamiblue.client.module.modules.render
+
+import net.minecraft.block.state.IBlockState
+import org.kamiblue.client.module.Category
+import org.kamiblue.client.module.Module
+import org.kamiblue.client.setting.settings.impl.collection.CollectionSetting
+
+internal object Xray : Module(
+    name = "Xray",
+    description = "Lets you see through blocks",
+    category = Category.RENDER
+) {
+    private val defaultVisibleList = linkedSetOf("minecraft:diamond_ore", "minecraft:iron_ore", "minecraft:gold_ore")
+
+    val showFluids by setting("Show Fluids", false)
```

Comment:
Why is this setting unused?

Reply:
Sorry was going to have that then changed my mind and forgot to remove it.

Reply:
Was going to have that setting, then realised that it wasn't needed and forgot to remove it.